src/components/views/read/cvForm/Education.tsx

```diff
@@ -18,105 +19,133 @@ import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs';
 import { DatePicker } from '@mui/x-date-pickers/DatePicker';
 import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider';
 
-// Import the default theme styles
-interface InnerObject {
-  [key: string]: string;
-}
+import { CVInfoObj, EducationInfoObj } from './types';
 
-interface ValuesObject {
-  [key: string]: InnerObject;
-}
-interface HandleModifyFunction {
-  (category: string, modifiedValues: any): void;
-}
 interface Props {
   nextPage: () => void;
   prevPage: () => void;
   nextStep: () => void;
   prevStep: () => void;
-  values: ValuesObject;
-  handleValues: HandleModifyFunction;
+  cvValues: CVInfoObj;
+  onCvValuesChange: (newCvValues: CVInfoObj) => void;
 }
 const Education: FC<Props> = ({
   nextPage,
   prevPage,
   nextStep,
   prevStep,
-  values,
-  handleValues,
+  cvValues,
+  onCvValuesChange,
 }) => {
   const handlePrev = (): void => {
     prevPage();
     prevStep();
   };
-  const [cards, setCards] = useState([{ id: 1 }]);
+  const [educationCards, setEducationCards] = useState<EducationInfoObj[]>([
+    {
+      id: 'card1',
+      degree: '',
+      institutionName: '',
+      major: '',
+      startDate: undefined,
+      endDate: undefined,
+      gpa: '',
+      country: '',
+    },
```

Comment:
TODO: in new issue.

Start with no cards, and put the "Add" button bellow the cards. It will be less weird than having to click the add button of the previous card to create a new card.